mysql-test/suite/rocksdb/t/collation.test

```diff
@@ -155,7 +155,7 @@ DROP TABLE t2;
 # test invalid regex (missing end bracket)
 --exec echo "" >$MYSQLTEST_VARDIR/log/mysqld.1.err
 SET GLOBAL rocksdb_strict_collation_exceptions="[a-b";
---exec grep -A 1 "Invalid pattern" $MYSQLTEST_VARDIR/log/mysqld.1.err | cut -d] -f2
+--exec grep "Invalid pattern" $MYSQLTEST_VARDIR/log/mysqld.1.err | cut -d] -f2
```

Comment:
Grepping the master error log file is flaky as its contents are arbitrary due to preceding testcases. The canonical MTR way would be to restart the server with a dedicated testcase error log file (and use one of the mysql-test/include files for the grep, i.e. search_pattern_in_file.inc)